pkg/controller/clusterinstallation/mysql.go

```diff
@@ -86,10 +86,13 @@ func (r *ReconcileClusterInstallation) getOrCreateMySQLSecrets(mattermost *matte
 			"DATABASE":      []byte("mattermost"),
 		}
 
-		return userPassword, r.createResource(mattermost, dbSecret, reqLogger)
+		return []string{"mmuser", userPassword, "mattermost"}, r.createResource(mattermost, dbSecret, reqLogger)
 	} else if err != nil {
 		reqLogger.Error(err, "Failed to check if mysql secret exists")
-		return "", err
+		return []string{}, err
```

Comment:
Can we always return an array of three strings, e.g. `[]string{"", "" ,""}`? Normally I'd say we should return nil for the first value any time there is an error but since line 89 could return a valid array with an error then I think it's better to be consistent and always return an array of length 3

Reply:
done